app/js/daemonManager.js

```diff
@@ -96,19 +100,13 @@ function DaemonManager() {
 		}, function() {
 			UI.notify('Starting siad...', 'start');
 
-			// daemon as a background process logs output to files
-			var out = Fs.openSync(Path.join(__dirname, siaPath, 'daemonOut.log'), 'w');
-			var err = Fs.openSync(Path.join(__dirname, siaPath, 'daemonErr.log'), 'w');
```

Comment:
I got an error once from these lines. Since siad is no longer a background process, these messages go to their parent process's console, e.g. the general UI's